benchmarks/scripts/plot_latest_expval_benchmarks.py

```diff
@@ -0,0 +1,67 @@
+import os
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+from datetime import datetime
+
+# Directory containing the CSV files
+directory = "../results"
+
+# Initialize a DataFrame to store combined data
+data = []
+
+# Process each CSV file
+for filename in os.listdir(directory):
```

Comment:
Style nit -- the glob approach above also works. Maybe better to use the same approach in all spots?

Reply:
Consistency is good https://github.com/unitaryfund/ucc/pull/168/commits/f6d634e09a74aca9f6a0d7338a55cb41f91b4300.